updating/updating-cluster.adoc

```diff
@@ -7,6 +7,18 @@ toc::[]
 
 You can update, or upgrade, an {product-title} cluster.
 
+[IMPORTANT]
+====
+During the {product-title} beta period, update functionality is limited:
+
+* You cannot update between different beta drops, such as from OpenShift 4 Beta
+Drop 2 to OpenShift 4 Beta Drop 3.
+* The ability to update from a beta drop of {product-title} to a full version of
```

Comment:
This drops off. I assume should either be "a full version of {product-title}" or just remove the "of".